### PR TITLE
BUG: ignore bugs instead of fixing them

### PR DESCRIPTION
**Pull request by jbrockmendel**
But at least enforce it elsewhere so we can prevent future bugs

Also a couple of typos in setup.cfg

### CODE REVIEW

File: statsmodels/sandbox/tsa/garch.py [hunk 1]
```diff
@@ -658,7 +658,16 @@ def gjrconvertparams(self, params, nar, nma):
...
-    return paramsclass
+    return paramsclass  # noqa:F821
+    # See GH#5756: there is no `paramsclass`, so this would raise a NameError
+    #  if ever hit.  But statsmodels convention does not allow us to change
+    #  this to the obviously-intended `return params2` because there is
+    #  no associated test.  There is not a reasonable way to _write_ an
+    #  associated test because this is not used anywhere.  Statsmodels
+    #  convention also does not allow deletion of unused, broken code,
+    #  so actually _fixing_ this error is not an option.
+    #  We therefore explicitly ignore it.
```
[the remark this answers is not in the document]
Comment (jbrockmendel) [reply statsmodels/sandbox/tsa/garch.py]: Is it inaccurate?

[the remark this answers is not in the document]
Comment (josef-pkt) [reply statsmodels/sandbox/tsa/garch.py]: We don't but everything that is not inaccurate in code comments.
E,g, we don't put on top of each file the sentence
"This module was changed during the cleanup crusage and might contain refactoring bugs or missing information. See github history for the original."
:)

[the remark this answers is not in the document]
Comment (jbrockmendel) [reply statsmodels/sandbox/tsa/garch.py]: Good point.  We could replace the entire function with `# There used to be an unfinished/broken function getgjrorsomething here.  See github history.`

Or for now I could shorten the comment to just `# See GH#5798` like the others.

**Comment by jbrockmendel** [comment]
azure fail unrelated

**Comment by bashtage** [comment]
Small PR with limited aims. Comments appropriate. LGTM.

**Pull request merged**